JSIL/AST/JSAstVisitor.cs

```diff
@@ -19,6 +19,21 @@ public struct EnclosingNode<T> {
             public string ChildName;
         }
 
+        public class NodeProcessedEventArg : EventArgs
```

Comment:
Here is common question - should we use standard event pattern (sender, eventArgs) - or could we optimize it.
In most scenarios I prefer use standard pattern as it simplify code readability for other developers.
Here, if you'd like it - let's change event signature to just `Action<JSNode>`